utility/src/main/java/org/oppia/android/util/parser/html/ImageTagHandler.kt

```diff
@@ -57,6 +60,15 @@ class ImageTagHandler(
         Spannable.SPAN_INCLUSIVE_EXCLUSIVE
       )
       output.replace(openIndex, output.length, spannableBuilder)
-    } else consoleLogger.w("ImageTagHandler", "Failed to parse $CUSTOM_IMG_ALT_TEXT_ATTRIBUTE")
+    } else consoleLogger.w(
+      "ImageTagHandler",
+      "Failed to parse $CUSTOM_IMG_ALT_TEXT_ATTRIBUTE"
+    )
+
+    if (!caption.isNullOrBlank()) {
+      output.append("\n").append(caption)
```

Comment:
Since this is being appended to the output, is it a seperate condition, or should it be a branch of the if condition above? Where is the output of the caption displayed(if at all). Please attach screenshots to the PR since this is a UI change.

Reply:
I attempted to implement the caption logic as a branch of the existing image handling condition, which serves as an alternative approach to the current implementation. However, the current setup, where the caption logic is handled separately, still produces the same results. I believe keeping the caption handling separate allows for more predictable alignment and styling, ensuring it doesn’t interfere with the image's layout. Nonetheless, I am open to exploring the alternative further if needed.

Reply:
That sounds good, thanks.